buildchow/templates/buildflow-template.yaml

```diff
@@ -0,0 +1,63 @@
+- job-template:
+    name: 'jbosstools-buildflow_{stream}'
+    disabled: '{obj:disabled}'
+    description: 'Build Flow that replaces the need for job cascading and interlinkage. Runs weekly. See <a href="https://issues.jboss.org/browse/JBIDE-13307">JBIDE-13307</a>'
+    jdk: '{default-jdk}'
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 10
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+    project-type: flow
+    dsl: |
+      parallel (
+      {{ ignore(FAILURE) {{ build("jbosstools-aerogear_{stream}") }} }},
+      {{ ignore(FAILURE) {{ build("jbosstools-base_{stream}") }} }}
+      )
+      parallel (
+      {{ 
+      parallel (
+      // {{ ignore(FAILURE) {{ build("jbosstools-gwt_{stream}") }} }},
+      {{ ignore(FAILURE) {{ build("jbosstools-forge_{stream}") }} }},
+      // {{ ignore(FAILURE) {{ build("jbosstools-freemarker_{stream}") }} }}, // unchanged for master branch so far
+      {{ 
+      ignore(FAILURE)  {{ build("jbosstools-server_{stream}") }}
+      parallel (
+      {{ ignore(FAILURE) {{ build("jbosstools-livereload_{stream}") }} }},
+      {{ ignore(FAILURE) {{ build("jbosstools-openshift_{stream}") }} }},
+      {{ ignore(FAILURE) {{ build("jbosstools-webservices_{stream}") }} }}
+        )
+        }}
+        ) 
+        }},
+        {{ 
+        parallel (
+        {{ 
+        ignore(FAILURE) {{ build("jbosstools-hibernate_{stream}") }}
+        // ignore(FAILURE) {{ build("jbosstools-birt_{stream}") }}
+      }},
+      {{
+      // Xulrunner is disabled - no need to build it
+      // build("xulrunner-1.9.2")
+      ignore(FAILURE) {{ build("jbosstools-vpe_{stream}") }}
+      ignore(FAILURE) {{ build("jbosstools-jst_{stream}") }}
+      ignore(FAILURE) {{ build("jbosstools-browsersim_{stream}") }}
+      }}
```

Comment:
Browsersim-standalone is missing. Aerogear now runs after browsersim and browsersim-standalone.